spec/api.spec.js

```diff
@@ -198,8 +218,8 @@ describe('API', function () {
           return Promise.resolve(specData)
         }
 
-        let tempSpec = { data: { ...specData, cluster: clusterDetail }, ...options }
-        clusterInfo = Object.assign({}, clusterDetail, { specData: tempSpec })
+        let tempSpec = { ...specData, ...options, masterIP: '192.168.1.1', credentials: SERVICE_ACCOUNT }
```

Comment:
just to verify my assumption, masterIP in this context is a fake value and not intended to point to anything real?

Reply:
Yep, totally fake for testing.